common/comms/transmission_adaptor.py

```diff
@@ -0,0 +1,9 @@
+import abc
+
+
+class TransmissionAdaptor(abc.ABC):
+
+    def make_request(self,  interaction_details, message):
```

Comment:
Shouldn't this be async def, or is that once @ZameerParvez has done his task (probably)? And shouldn't this method return some response? And this method probably should have a `abstractmethod` decorator.